test/data/plugin/bundled/MANIFEST.MF

```diff
@@ -0,0 +1,393 @@
+ 0.1.jar lib/dictzip-0.13.0.jar lib/annotations-23.0.0.jar lib/protobuf-
+ java-3.20.0.jar lib/caffeine-2.9.3.jar lib/jna-platform-5.6.0.jar lib/j
+ sch.agentproxy.connector-factory-0.0.7.jar lib/jsch.agentproxy.usocket-
+ jna-0.0.7.jar lib/jsch.agentproxy.pageant-0.0.7.jar lib/jna-5.6.0.jar l
+ ib/morfologik-fsa-builders-2.1.0.jar lib/morfologik-speller-2.1.0.jar l
+ ib/morfologik-polish-2.0.0.jar lib/morfologik-stemming-2.1.0.jar lib/mo
+ rfologik-fsa-2.1.0.jar lib/hppc-0.7.1.jar lib/annotations-12.0.jar lib/
+ lucene-backward-codecs-5.2.1.jar lib/berkeleylm-1.1.2.jar lib/JavaEWAH-
+ 1.1.13.jar lib/jzlib-1.1.1.jar lib/junixsocket-native-common-2.4.0.jar 
+ lib/junixsocket-common-2.4.0.jar lib/eddsa-0.3.0.jar lib/sequence-libra
+ ry-1.0.4.jar lib/sqljet-1.1.15.jar lib/jsch.agentproxy.svnkit-trilead-s
+ sh2-0.0.7.jar lib/trilead-ssh2-1.0.0-build222.jar lib/lz4-java-1.4.1.ja
+ r lib/beansbinding-1.2.1.jar lib/diffutils-1.2.1.jar lib/groovy-ant-3.0
+ .9.jar lib/groovy-astbuilder-3.0.9.jar lib/groovy-cli-picocli-3.0.9.jar
+  lib/groovy-groovysh-3.0.9.jar lib/groovy-console-3.0.9.jar lib/groovy-
+ datetime-3.0.9.jar lib/groovy-groovydoc-3.0.9.jar lib/groovy-docgenerat
+ or-3.0.9.jar lib/groovy-jmx-3.0.9.jar lib/groovy-json-3.0.9.jar lib/gro
+ ovy-jsr223-3.0.9.jar lib/groovy-macro-3.0.9.jar lib/groovy-nio-3.0.9.ja
+ r lib/groovy-servlet-3.0.9.jar lib/groovy-sql-3.0.9.jar lib/groovy-swin
+ g-3.0.9.jar lib/groovy-templates-3.0.9.jar lib/groovy-test-3.0.9.jar li
+ b/groovy-test-junit5-3.0.9.jar lib/groovy-testng-3.0.9.jar lib/groovy-x
+ ml-3.0.9.jar lib/groovy-3.0.9.jar lib/checker-qual-3.19.0.jar lib/error
+ _prone_annotations-2.10.0.jar lib/jsonic-1.2.11.jar lib/openregex-1.1.1
+ .jar lib/guava-16.0.1.jar lib/antlr-runtime-3.4.jar lib/jsch.agentproxy
+ .usocket-nc-0.0.7.jar lib/jsch.agentproxy.sshagent-0.0.7.jar lib/jsch.a
+ gentproxy.core-0.0.7.jar lib/opennlp-tools-1.6.0.jar lib/opennlp-tokeni
+ ze-models-1.5.jar lib/opennlp-postag-models-1.5.jar lib/opennlp-chunk-m
+ odels-1.5.jar lib/jwordsplitter-4.1.jar lib/cjftransform-1.0.1.jar lib/
+ morphology-el-1.0.0.jar lib/ant-junit-1.10.11.jar lib/ant-1.10.11.jar l
+ ib/ant-launcher-1.10.11.jar lib/ant-antlr-1.10.11.jar lib/picocli-4.6.1
+ .jar lib/qdox-1.12.1.jar lib/javaparser-core-3.23.0.jar lib/jline-2.14.
+ 6.jar lib/junit-4.13.2.jar lib/junit-platform-launcher-1.7.2.jar lib/ju
+ nit-platform-engine-1.7.2.jar lib/junit-platform-commons-1.7.2.jar lib/
+ junit-jupiter-engine-5.7.2.jar lib/junit-jupiter-api-5.7.2.jar lib/test
+ ng-7.4.0.jar lib/platform-3.4.0.jar lib/hamcrest-core-1.3.jar lib/opent
+ est4j-1.2.0.jar lib/jcommander-1.78.jar lib/jquery-3.5.1.jar
+
+Name: org/omegat/core/dictionaries/
+Plugin-Name: Dictionary driver[bundle]
+Plugin-Category: dictionary
+Plugin-Description: Bundled dictionary files connectors. There are two b
+ undled connectors. LingvoDSL, connector  for .dsl, .dsl.dz, .dsl.gz and
+  StarDict, connector for .ifo, *.dict
+
```

Comment:
Now it can handle package section of MANIFEST.MF

You can see an article 
https://www.baeldung.com/java-jar-manifest#sections

> In addition, a header appearing in a per-entry section overrides the same header in the main section. It is common for per-entry sections to contain information on package versions and sealing plus digital signing.

We can use the nature to "override", we can provide multiple Plugin-Category, Plugin-Descriptions in single manifest file.